main.go

```diff
@@ -132,6 +132,7 @@ func main() {
 	log.Infoln("Build context", version.BuildContext())
 
 	// Bail early if the config is bad.
+	var err error
```

Comment:
In the case of multiple returns, `:=` will only create if necessary.  I've seen the compile error if the declare is used in a `switch` or `if` and there are uses outside of the context.  For those you need to declare with a `var` outside the switch/if blocks.